packages/invoices/src/OutboundShipment/DetailView/ContentArea.tsx

```diff
@@ -107,7 +107,7 @@ export const ContentAreaComponent: FC<ContentAreaProps> = ({ onRowClick }) => {
   if (!rows) return null;
 
   return (
-    <Box flexDirection="column">
+    <Box flexDirection="column" style={{ width: '100%' }}>
```

Comment:
was not stretching full width on larger screens

Reply:
How wide are we talking? I ran in full-screen on 1680x1050 with the Drawer closed and the table still went full-width without this style setting. 

Then again, I guess there's no downside either way, so... 🤷‍♂️

Reply:
my monitor is 1920 and doesn't feel like super-wide, but does this:

![Screen Shot 2022-04-12 at 2 16 04 PM](https://user-images.githubusercontent.com/9192912/162865899-215a62c9-f4fc-4c6a-8cf9-e92a127c1e86.png)

The problem was original noticed by Richard on his screen in the office. probably similar?